core/rend/gles/gltex.cpp

```diff
@@ -364,119 +394,496 @@ struct TextureCacheData
 	}
 };
 
-#include <map>
 map<u64,TextureCacheData> TexCache;
 typedef map<u64,TextureCacheData>::iterator TexCacheIter;
 
 //TexCacheList<TextureCacheData> TexCache;
 
 struct FBT
 {
-	u32 TexAddr;
-	GLuint depthb,stencilb;
+	u32 texAddress;
+	u16 texData[1024*1024];
```

Comment:
Can we avoid this by writing directly to vram on readouts?

Reply:
Actually, to improve performance RTT is implemented in a way that vram is not used (because vram is not valid in the next frame), so I need to store a texture for future use. I wrote about it in this comment (third paragraph): https://github.com/reicast/reicast-emulator/pull/1460#issuecomment-439205071